LayoutTests/fast/gradients/css3-gradient-parsing.html

```diff
@@ -9,67 +9,111 @@
 
 <script type="text/javascript">
 
-function testGradient(css, queryProp)
+function testGradientSpecifiedValue(css, queryProp)
 {
     var div = document.createElement('div');
     div.setAttribute('style', css);
     document.body.appendChild(div);
 
-    var result = getComputedStyle(div).getPropertyValue(queryProp);
+    var result = div.style.getPropertyValue(queryProp);
     document.body.removeChild(div);
     return result;
 }
 
-function testSame(dir, prop, altProp, value)
+function testGradientComputedValue(css, queryProp)
```

Comment:
Starting to wish these tests were in Web Platform Tests. I find it easier to make a good test in our own tests, but the power to drive interoperability when the tests are in WPT is so worthwhile. Maybe we can get someone to convert these?

Reply:
I am just putting it on my list of things to upstream. There is a whole parsing section in the css-images WPT folder that I already have contributed a bunch of test to.